Lib/test/test_syntax.py

```diff
@@ -625,7 +629,6 @@
     >>> def f(*xx, __debug__): pass
     Traceback (most recent call last):
     SyntaxError: cannot assign to __debug__
-
```

Comment:
Sorry for the nit: Could you restore this empty line?

Reply:
done